Makefile

```diff
@@ -780,6 +780,13 @@ ifeq ($(CI),true)
 	cp /build/* $(DIST)/binaries
 endif
 
+.PHONY: release-freebsd
```

Comment:
```suggestion
.PHONY: release-freebsd-amd64
```

Reply:
I'd prefer to keep as-is for when arm64 support for freebsd comes it'd be via this make target, just like how it was done with darwin.